doc/source/examples/devportal.rst

```diff
@@ -0,0 +1,41 @@
+.. _ref_devportal_examples:
+
+
+Developer portal articles
+=========================
+
+In this section, you'll find informative and insightful blog
+articles crafted by our expert developers at Ansys and published
+in the `Developer Portal <developer_portal_>`_.
+Whether you're a seasoned pro or just getting started with PYMAPDL, our blog
+articles cover a wide range of topics, from best practices and tips to in-depth
+tutorials and real-world use cases. Our aim is to provide you with a valuable
+resource that will help you harness the full potential of PYMAPDL for your
+engineering and simulation needs.
+
+If you have questions or suggestions regarding other Ansys products, you can
+also visit the `Developer Portal <developer_portal_>`_. It's your hub for all
+things related to our products, where you can connect with our experts and
+engage with our growing community.
+
+.. Limiting the amount of lines to 2 at 80 chars
+.. grid:: 2
+    :gutter: 2
+
+    .. grid-item-card:: Biomedical Catheter Design Analysis
+      :img-top: https://developer.ansys.com/sites/default/files/inline-images/BMCatheter_1_0.png
+      :link: https://developer.ansys.com/blog/biomedical-catheter-design-analysis
+      :text-align: center
+      :class-title: pyansys-card-title
+
+      In this article, we explore the mechanical behavior of a flexible,
+      biomedical catheter under large deformation. Using the scripting...
```

Comment:
```suggestion
      This article explores the mechanical behavior of a flexible,
      biomedical catheter under large deformation. Using the scripting...
```
Did you want the description to trail off? Wouldn't it be better to use only 
the single-sentence description?

Reply:
I just didn't want to come up with a summary 😅